### ISSUE TITLE
how to pretrain the LPD-net

### ISSUE THREAD

**Issue by Everbeast** [qiaozhijian/VCR-Net#6]
I've trained the lpd-net without changing any parameters, but I can not get the model as good as you have gived in "pretrained/lpd-pretrained.t7"
here is the test result:
result of my try:
EPOCH:: -1, Loss: 0.086327, test_LossPose: 0.141541, Cycle Loss: 0.000000, MSE: 0.086327, RMSE: 0.293815, MAE: 0.230498, rot_MSE: 510.304260, rot_RMSE: 22.589914, rot_MAE: 17.948847, trans_MSE: 0.040334, trans_RMSE: 0.200833, trans_MAE: 0.170415

result of "pretrained/lpd-pretrained.t7"
EPOCH:: -1, Loss: 0.044717, test_LossPose: 0.031803, Cycle Loss: 0.000000, MSE: 0.044717, RMSE: 0.211464, MAE: 0.149825, rot_MSE: 88.623634, rot_RMSE: 9.414013, rot_MAE: 6.795428, trans_MSE: 0.014510, trans_RMSE: 0.120459, trans_MAE: 0.091989

**Comment by qiaozhijian** [comment]
Try changing some parameters, such as the learning rate.
But step back, if you use your own pre-training model, will the end result also be worse?
